draft-ietf-quic-manageability.md

```diff
@@ -167,13 +163,15 @@ The following information is exposed in QUIC packet headers:
   Header Form and fixed bits. Header types correspond to stages of the
   handshake; see Section 17.2 of {{QUIC-TRANSPORT}} for details.
 
-- version number: the version number present in the long header, and identifies
-  the version used for that packet. Note that during Version Negotiation (see
-  {{version}}, and Section 17.2.1 of {{QUIC-TRANSPORT}}, the version number
-  field has a special value (0x00000000) that identifies the packet as a Version
-  Negotiation packet. QUIC versions that start with 0xff are IETF drafts. QUIC
-  versions that start with 0x0000 are reserved for IETF consensus documents,
-  for example the QUIC version 1 is expected to use version 0x00000001.
+- version number: the version number is present in the long header, and
+  identifies the version used for that packet. Note that during Version
+  Negotiation (see {{version}} and Section 17.2.1 of {{QUIC-TRANSPORT}}), the
+  version number field has a special value (0x00000000) that identifies the
+  packet as a Version Negotiation packet. QUIC versions that start with 0xff are
+  IETF drafts. QUIC versions that start with 0x0000 are reserved for IETF
+  consensus documents. For example, QUIC version 1 uses version 0x00000001.
```

Comment:
Are these convention still true? I guess we don't have that in the transport spec and therefore should probably also remove it here...?

Reply:
How about "Many QUIC versions that start with 0xff"?  This is a statement of fact, though you're right that in the future other stuff could be in this space.

Reply:
Yep that's probably good to note!